### ISSUE TITLE
incorrect package.json definition for use of `ajv8`

### ISSUE THREAD

**Issue by heath-freenome** [ajv-validator/ajv-formats#80]
We are seeing an [issue](https://github.com/rjsf-team/react-jsonschema-form/issues/3413) in @rjsf where the user is getting duplicate `ajv` instances in their webpack distribution. I'm fairly certain it is because `ajv-formats` has an incorrect `dependencies` for `ajv8` since it is also a `peerDependency` AND `devDependency`. It seems like the `dependency` itself is causing this issue. Moreover, it probably make sense to not have ajv8 as an optional peerDependency once you remove it as a dependency

**Comment by heath-freenome** [comment]
@epoberezkin I can create a PR to fix this if this would help speed this along

**Comment by reedoso** [comment]
@heath-freenome 
I agree with this. Having `ajv` both as a dependency and a peer dependency feels unnecessary and can easily lead to duplicate installs or bundle duplication.

For a plugin like `ajv-formats`, it seems cleaner to keep `ajv` as a peer dependency and use it only as a dev dependency for tests and development.

**Comment by reedoso** [comment]
Pull Request: https://github.com/ajv-validator/ajv-formats/pull/121
@epoberezkin Please check it out
